srcpkgs/gnome-core/template

```diff
@@ -0,0 +1,115 @@
+# Template file for 'gnome-core'
+pkgname=gnome-core
+version=3.36.0
+revision=1
+build_style=meta
+short_desc="GNOME meta-package for Void Linux - core components"
+maintainer="Enno Boland <gottox@voidlinux.org>"
+license="GPL-2.0-only, LGPL-2.0-only"
+homepage="https://www.gnome.org"
+
+depends="
+ NetworkManager>=1.22.10_2
+ adwaita-icon-theme>=3.36.1_1
+ at-spi2-atk>=2.34.1_1
+ at-spi2-core>=2.34.0_1
+ atk>=2.34.1_1
+ atkmm>=2.28.0_1
+ cairomm>=1.12.2_3
+ cantarell-fonts>=0.201_1
+ caribou>=0.4.21_3
+ clutter>=1.26.4_1
+ clutter-gst3>=3.0.27_2
+ clutter-gtk>=1.8.4_4
+ cogl>=1.22.6_1
+ dconf>=0.36.0_1
+ evolution-data-server>=3.36.2_1
+ file-roller>=3.36.2_1
+ folks>=0.14.0_1
+ font-adobe-source-code-pro>=2.030R_2
+ gcab>=1.4_1
+ gcr>=3.36.0_1
+ gdk-pixbuf>=2.40.0_2
+ gdm>=3.36.2_1
+ gedit>=3.36_2.1
+ geocode-glib>=2.64.2_1
+ gfbgraph>=0.2.3_6
+ gjs>=1.65.2_1
+ glib>=2.64.2_1
+ glib-networking>=2.64.1_2
+ glibmm>=2.62.0_1
+ gmime>=2.6.23_3
+ gnome-autoar>=0.2.4_1
+ gnome-backgrounds>=3.36.0_2
+ gnome-bluetooth>=3.34.1_1
+ gnome-color-manager>=3.36.0_1
+ gnome-control-center>=3.36.1_1
+ gnome-desktop>=3.36.2_1
+ gnome-getting-started-docs>=3.36.2_1
+ gnome-initial-setup>=3.36.2_1
+ gnome-keyring>=3.36.0_1
+ gnome-menus>=3.36.0_1
+ gnome-online-accounts>=3.36.0_1
+ gnome-online-miners>=3.34.0_1
+ gnome-session>=3.36.0_1
+ gnome-settings-daemon>=3.36.1_1
+ gnome-shell>=3.36.2_1
+ gnome-terminal>=3.36.1.1_1
+ gnome-themes-extra>=3.28_2
+ gnome-user-docs>=3.36.2_1
+ gnome-video-effects>=0.4.3_2
+ gobject-introspection>=1.64.1_1
+ gom>=0.3.3_4
+ grilo>=0.3.12_1
+ grilo-plugins>=0.3.11_1
+ gsettings-desktop-schemas>=3.36.0_2
+ gsound>=1.0.2_6
+ gspell>=1.8.3_1
+ gssdp>=1.2.2_1
+ gtk+3>=3.24.20_1
+ gtkmm>=3.24.1_2
+ gtksourceview>=3.24.11_1
+ gupnp>=1.2.2_1
+ gupnp-av>=0.12.11_4
+ gupnp-dlna>=0.10.5_3
+ gupnp-igd>=0.2.5_3
+ gvfs>=1.44.1_1
+ json-glib>=1.4.4_3
+ libcroco>=0.6.13_1
+ libgdata>=0.17.12_1
+ libgee08>=0.20.3_1
+ libgepub>=0.6.0_3
+ libgnomekbd>=3.26.1_2
+ libgsf>=1.14.46_1
+ libgtop>=2.40.0_2
+ libgweather>=3.34.0_1
+ libgxps>=0.3.1_3
+ libmediaart>=1.9.4_3
+ libnotify>=0.7.7_3
+ libpeas>=1.26.0_1
+ librsvg>=2.47.1_1
+ libsecret>=0.20.2_1
+ libsigc++>=2.10.3_1
+ libsoup>=2.70.0_1
+ libxml2>=2.9.10_2
+ libxslt>=1.1.34_2
+ libzapojit>=0.0.3_9
+ mm-common>=1.0.0_1
+ mousetweaks>=3.32.0_2
+ mutter>=3.36.2_1
+ nautilus>=3.36.2_1
+ network-manager-applet>=1.8.24_1
+ orca>=3.36.2_1
+ pango>=1.42.4_7
+ pangomm>=2.42.0_1
+ phodav>=2.3_2
+ python-atspi>=2.36.0_2
+ python-gobject>=3.36.0_1
+ rest>=0.8.1_3
+ tracker>=2.3.4_1
+ vala>=0.46.3_1
```

Comment:
Doesn it need vala, and yeld-tools?
I think it only requires libvala,

Reply:
Sorry for the late reply. 

Deciding what to add in `gnome-core` and what not is pretty subjective and hence open to discussion. As a reference GNOME considers these as "core" https://ftp.gnome.org/pub/GNOME/core/3.36/3.36.2/sources/. packages, which I've tried to replicate with `gnome` meta-package.

Personally, I am also in favor of moving these two to `gnome`.